web/.ncurc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "reject": ["buffer", "monaco-editor"]
```

Comment:
Remove "monaco-editor" now that the bug went away that was requiring an earlier version